onyo/lib/commands.py

```diff
@@ -401,16 +402,21 @@ def onyo_edit(inventory: Inventory,
     paths
         Paths of assets to edit.
     message
-        A custom commit message.
+        Commit message to append to the auto-generated message.
+    auto_message
+        If `True`, do auto-generate a commit message subject line.
+        The default can be configured via 'onyo.commit.auto-message'.
```

Comment:
Nit: "can be" -> is

IMO, this should more explicitly state that `None` means use the default that's configured at 'onyo.commit.auto-message'.